libr/bin/format/mdmp/mdmp_pe.c

```diff
@@ -187,32 +189,30 @@ RList *PE_(r_bin_mdmp_pe_get_sections)(struct PE_(r_bin_mdmp_pe_bin) *pe_bin) {
 			ptr->perm |= R_PERM_W;
 		}
 		if (R_BIN_PE_SCN_IS_READABLE (sections[i].perm)) {
-			ptr->perm |= R_PERM_X;
+			ptr->perm |= R_PERM_R;
 		}
 		if (R_BIN_PE_SCN_IS_SHAREABLE (sections[i].perm)) {
 			ptr->perm |= R_PERM_SHAR;
 		}
-#define X 1
-#define ROW (4 | 2)
-		if (ptr->perm & ROW && !(ptr->perm & X) && ptr->size > 0) {
-			if (!strcmp (ptr->name, ".rsrc") ||
-				!strcmp (ptr->name, ".data") ||
-				!strcmp (ptr->name, ".rdata")) {
-					ptr->is_data = true;
-				}
+		if ((ptr->perm & R_PERM_R) && !(ptr->perm & R_PERM_X) && ptr->size > 0) {
```

Comment:
This block is where the fix is, the rest of the code is from formatting fixes by `indent.sh`